src/ucp/core/ucp_types.h

```diff
@@ -40,7 +40,7 @@ UCP_UINT_TYPE(UCP_MD_INDEX_BITS)     ucp_md_map_t;
 
 
 /* Lanes */
-#define UCP_MAX_LANES                16
+#define UCP_MAX_LANES                64
```

Comment:
we should probably change code that uses ucs_ffs32 on the lane map to use ucs_ffs64
for example in protov2 rndv/put/zcopy

Reply:
I changed all the entries that I found, but I haven't found one in rndv/put/zcopy. Maybe they were removed in https://github.com/openucx/ucx/pull/9862